src/managers/GuildMemberManager.js

```diff
@@ -149,6 +150,47 @@ class GuildMemberManager extends BaseManager {
     return data.reduce((col, member) => col.set(member.user.id, this.add(member, cache)), new Collection());
   }
 
+  /**
+   * Edits a member of the guild.
+   * <info>The user must be a member of the guild</info>
+   * @param {UserResolvable} member The member to edit
```

Comment:
```suggestion
   * @param {UserResolvable} user The member to edit
```

Reply:
Shouldn't I actually make this GuildMemberResolvable?

Reply:
I'm not sure, since those types are identical. But `GuildMemberResolvable` might make more sense here.
The same applies to the `kick` method then too.